css/device-catalog.css

```diff
@@ -0,0 +1,884 @@
+}
+.button-search {
+    display: none;
+}
+.input-search:focus + button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 85px;
+    height: 49px;
+    border: 2px solid #000000;
+    background-color:#f7e296;
+}
+.header-buttons .button-search:hover {
+    background: #000000;
+    color: #ffffff;
+}
+.header-buttons .button-search:active {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 85px;
+    height: 49px;
+    background: #000000;
+    color: rgba(255, 255, 255, 0.3);
+}
+.user-buttons {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+
+    width: 50%;
+    margin-left: 20px;
+    
+    font-family: 'Opensanslight', 'sans-serif';
+    font-size: 15px;
+    line-height: 30px;
+}
+.user-buttons li:first-child {
+    margin-right: 19px;
+}
+.user-buttons li:nth-child(2) {
+    margin-right: auto;
+    opacity: 0.3;
+}
+.user-buttons li:nth-child(3) {
+    margin-right: 37px;
+}
+.user-buttons a {
+    position: relative;
+}
+.user-header {
+    padding-left: 26px;
+}
+.user-header::before {
+    content: url(.//img/user.svg);
+    position: absolute;
+    left: 0px;    
+}
+.button-compare {
+    padding-left: 20px;
+}
+.button-compare::before {   
+    content: url(.//img/compare.svg); 
+    position: absolute;  
+    left: 0px;   
+}
+.button-basket {
+    padding-left: 23px;
+}
+.button-basket::before {
+    content: url(.//img/cart.svg);
+    position: absolute;
+    left: 0px;  
+}
+.user-buttons a:hover,
+.nav-list a:hover {
+    opacity: 0.6;
+}
+.user-buttons a:active,
+.nav-list a:active {
+    opacity: 0.3;
+}
+.nav {
+    display: flex;
+    position: relative;
+    align-items: center;
+    justify-content: flex-start;
+    width: auto;
+    margin-left: 60px;
+    margin-right: 60px;
+}
+.nav-list {
+    display: flex;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+    width: 100%;
+}
+.nav-link {
+    font-size: 18px;
+    line-height: 24px;
+    text-transform: uppercase;
+    letter-spacing: 3px;
+}
+.nav-catalog {
+    width: 50%;
+}
+.nav-delivery {
+    margin-right: 35px;
+    margin-left: 20px;
+}
+.nav-warranty {
+    margin-right: auto;
+}
+.nav-catalog-link {
+    position: relative;
+    padding-right: 28px;
+}
+.nav-catalog-link::after {
+    content: url(.//img/plus.svg);
+    position: absolute;
+    right: 0px;
+    top: 2px;
+    opacity: 0.3;    
+}
+
+.dropdown-menu {
+    display: none;
+    position: absolute;
+
+    min-width: 1160px;
+    padding-left: 60px;
+    padding-top: 40px;
+    padding-bottom: 20px;
+    left: -60px;
+    top: 21px;
+
+    background-color: #f7e296;
+    z-index: 2;
+
+    font-family: 'Opensanslight', 'sans-serif';
+    font-size: 15px;
+    line-height: 36px;
+}
+.dropdown-menu div {
+    display: flex;
+    flex-direction: column;
+    margin-right: 40px;
+}
+.nav-catalog-link:hover ~ .dropdown-menu {
+    display: flex;
+}
+.dropdown-menu:hover {
```

Comment:
really need this rule? doesn't the rule above do the same?

Reply:
Yes, I do. Because the rule above makes dromdown-menu visiable but only when you hover on the link. But it will disappear as you move cursour out of the link. So dropdown-menu:hover needs to see the menu and work with it.